engine/src/image.rs

```diff
@@ -8,18 +8,48 @@ use super::svg::SvgTangibleObject;
 pub struct Image {
     path: String,
     round: Option<usize>,
+    /// (offset(x, y), stdDeviation)
+    shadow: Option<((usize, usize), usize)>,
 }
 
 impl Image {
     pub fn new_from_path(path: String) -> Self {
-        Self { path, round: None }
+        Self {
+            path,
+            round: None,
+            shadow: None,
+        }
+    }
+
+    /// Get padding size.
+    ///
+    /// Padding size is the extra space around the image,
```

Comment:
nitpick: might be worth calling out that the return value is on x and y and we treat both side on one direction the same spacing